sound/soc/sof/topology.c

```diff
@@ -1441,12 +1455,15 @@ static int sof_widget_load_dai(struct snd_soc_component *scomp, int index,
 
 	/* configure dai IPC message */
 	memset(&comp_dai, 0, sizeof(comp_dai));
+
+	/* get generic component configuration from the template */
```

Comment:
You changed the variable name but kept the old name "template" in the comment.

Reply:
the comp is the "template", no need to change in the comment?

Reply:
If "template" was a good name, then why was the parameter renamed to "comp"?

This comment could be "initialize the base type". Assuming designated initializers and good names there's no need for any comment.

Reply:
I am not a guy torn on comment naming, maybe "from the comp template" sounds better for you @marc-hb ?

Reply:
> I am not a guy torn on comment naming,

“Programs must be written for people to read, and only incidentally for machines to execute.”